### PR TITLE
daily coding Practice 7 added

### PR DESCRIPTION
**Pull request by waghmaresuyog**

### CODE REVIEW

File: simple/src/codingPractice/SwapLatter.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    public static void main(String[] args) {
+
+        String swapName= "suyog waghmare";
+        System.out.println("the Original string is "+swapName);
+        SwapLatter swapLatter = new SwapLatter();
+
+        System.out.println(swapLatter.swap(swapName,0,7));
+    }
+    public  char [] swap(String swapName,int indexOne, int indexTwo)
+    {   char charSwap[] =swapName.toCharArray();
```
Comment (deepakghadge19) [inline simple/src/codingPractice/SwapLatter.java:13]: Format the code

File: simple/src/codingPractice/SwapLatter.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package codingPractice;
+
+public class SwapLatter {
+    public static void main(String[] args) {
+
```
Comment (deepakghadge19) [inline simple/src/codingPractice/SwapLatter.java:5]: No need of space here

File: simple/src/codingPractice/SwapLatter.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        SwapLatter swapLatter = new SwapLatter();
+
+        System.out.println(swapLatter.swap(swapName,0,7));
+    }
+    public  char [] swap(String swapName,int indexOne, int indexTwo)
+    {   char charSwap[] =swapName.toCharArray();
+        char tempChar = charSwap[indexOne];
+        charSwap[indexOne]=charSwap[indexTwo];
+        charSwap[indexTwo]=tempChar;
+
```
Comment (deepakghadge19) [inline simple/src/codingPractice/SwapLatter.java:17]: No need of sapce

File: simple/src/codingPractice/SwapLatter.java (hunk 1 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/SwapLatter.java]: done

File: simple/src/codingPractice/SwapLatter.java (hunk 3 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/SwapLatter.java:12]: done

**Review by deepakghadge19 — APPROVED** [review]

**Review by Vrushalee1234 — APPROVED** [review]

**Pull request merged**